RiotSharp/Endpoints/Interfaces/Static/IStaticDataEndpoint.cs

```diff
@@ -0,0 +1,18 @@
+﻿using RiotSharp.Endpoints.Interfaces.Static;
+
+namespace RiotSharp.Endpoints.Interfaces.Static
+{
+    public interface IStaticDataEndpoint
+    {
```

Comment:
You might rename this to `IStaticDataEndpointCollection` or `IStaticDataEndpoints`, so it's distinguishable from the actual endpoints.